plugin/src/main/resources/io/jenkins/plugins/coverage/model/visualization/dashboard/CoverageColumn/column.jelly

```diff
@@ -1,27 +1,30 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core">
 
+  <link rel="stylesheet" href="${resURL}/plugin/code-coverage-api/css/column-style.css"/>
+
   <j:set var="coverageText" value="${it.getCoverageText(job)}"/>
   <j:set var="coverageValue" value="${it.getCoverageValue(job)}"/>
 
   <j:choose>
     <j:when test="${coverageValue.isPresent()}">
       <j:set var="displayColors" value="${it.getDisplayColors(job, coverageValue)}"/>
-      <j:set var="lineColor" value="${displayColors.getLineColorAsHex()}"/>
       <j:set var="backgroundColor" value="${displayColors.getFillColorAsHex()}"/>
       <j:set var="url" value="${it.getRelativeCoverageUrl(job)}"/>
       <j:set var="fillPercentage" value="${it.getBackgroundColorFillPercentage(coverageText)}"/>
-      <td style="color: ${lineColor};
-            background-image: linear-gradient(90deg, ${backgroundColor} ${fillPercentage}, transparent ${fillPercentage});"
-          align="left">
-        <j:choose>
-          <j:when test="${url.isEmpty()}">
-            ${coverageText}
-          </j:when>
-          <j:otherwise>
-            <a style="color:${lineColor}" href="${rootURL}/${job.url}lastSuccessfulBuild/${url}">${coverageText}</a>
-          </j:otherwise>
-        </j:choose>
+      <td align="left">
+        <div class="coverage-column-outer">
+          <div class="coverage-column-inner" style="background-image: linear-gradient(90deg, ${backgroundColor} ${fillPercentage}, transparent ${fillPercentage});">
+            <j:choose>
+              <j:when test="${url.isEmpty()}">
+                ${coverageText}
+              </j:when>
+              <j:otherwise>
+                <a style="color:${lineColor}" href="${rootURL}/${job.url}lastSuccessfulBuild/${url}">${coverageText}</a>
```

Comment:
`style="color:${lineColor}"` can be removed if there should always be the default URL color in use.

Reply:
That's right. I just deleted the variable and forgot to remove the reference.